src/contrib/serializers/Akka.Serialization.TestKit/AkkaSerializationSpec.cs

```diff
@@ -230,7 +230,7 @@ public void CanSerializeTailChoppingPool()
 
             var supervisor = new OneForOneStrategy(decider);
 
-            var message = new TailChoppingPool(10, null, supervisor, "abc", TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(2));
+            var message = new TailChoppingPool(10, null, TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(2), supervisor, "abc");
```

Comment:
We should probably change it back - better off having backwards compat for current users